python/triton/language/semantic.py

```diff
@@ -1208,10 +1208,39 @@ def mfma_supported(M, N, K, allow_tf32, ret_scalar_ty, target) -> bool:
         return False
     return True
 
+def mfma_supported_granularity(m, n, k) -> bool:
```

Comment:
maybe it should look like this?
```suggestion
def wmma_supported_granularity(m, n, k) -> bool:
```